### ISSUE TITLE
Service status block links use 'Read more' as link text

### ISSUE THREAD

**Issue by danchamp** [localgovdrupal/localgov_services#60]
The service status block lists and links to updates using the link text 'Read more'. This lacks context for screen reader users, and if multiple updates are present it results in multiple links with the same link text linking to different destinations:

https://github.com/localgovdrupal/localgov_services/blob/cc3341ce47953dea11b77e7b8856e6a0f11588f0/modules/localgov_services_status/templates/service-status-block.html.twig#L11-L18

<img width="826" alt="Screenshot 2020-07-02 at 15 16 58" src="https://user-images.githubusercontent.com/261421/86370053-2076ad00-bc77-11ea-9f34-e492874da785.png">

This should be improved by either adding an aria-describedby attribute referencing the status title, or by extending the link text with screen reader accessible text containing the status title.
